packages/lib/pod/src/podTypes.ts

```diff
@@ -1,4 +1,8 @@
-import { r as BabyJubjubR } from "@zk-kit/baby-jubjub";
+// Prime order of the alt_bn128 curve.
+// Should always be equal to the `r` constant in @zk-kit/baby-jubjub.
+export const BabyJubjubR = BigInt(
```

Comment:
This doesn't need to be exported (and wasn't before).  It just needs to be available locally for the definition of POD_CRYPTOGRAPHIC_MAX.

Reply:
Oh, it has to be exported to be testable.  I hate the conflation of exported-from-module and exported-from-package in TypeScript.
It could be stuck in a separate constants file to accomplish making it package-private?  That may be overkill.  If it's exported, I think it should be named in ALL_CAPS like the other constants.

Reply:
I've gone with this approach, which does indeed seem simpler and avoids the unnecessary export.